src/node_process_methods.cc

```diff
@@ -604,7 +604,9 @@ static void LoadEnvFile(const v8::FunctionCallbackInfo<v8::Value>& args) {
 
   Dotenv dotenv{};
 
-  switch (dotenv.ParsePath(path)) {
+  // TODO(dario-piotrowicz): update `process.loadEnvFile` to also accept
+  // sections
```

Comment:
Any chance this todo can be completed as part of this? I'd rather avoid the inconsistency, even if only temporary.

Reply:
Of course, I was just trying to keep the scope of the PR smaller, I will do that here as well :slightly_smiling_face::+1:

Reply:
Done :smile: